### ISSUE TITLE
Choose the follow-up email address from an answer

### ISSUE THREAD

**Issue by titidelaff** [pluginsGLPI/formcreator#1098]
Hello,

I wish I could choose the follow-up email address from an answer.

**Comment by btry** [comment]
Hi

Formcreator does not chooses itself the email address for notifications. It is done by GLPI. I think this can be solved when the user customizes his preferences.

**Comment by titidelaff** [comment]
Maybe add this feature on futur update?

**Comment by btry** [comment]
I'm afraid not because I don't see a way for plugins to choose the recipient email address for users having several ones. This needs investigation in the source code to be sure. 

I mark the issue as feature request, but it is still unplanned for now.

**Comment by titidelaff** [comment]
![image](https://user-images.githubusercontent.com/23405363/45280637-75ab6280-b4d5-11e8-8300-d0a25536d732.png)
In a normal ticket, here is how the choice of the mail destined for the follow-up is presented.

**Comment by btry** [comment]
Indeed. If we can do this from a regular ticket then it is possible to develop the feature.

the feature needs the following developments : 
- provide a new field for actors added in the target settings
- use a question to set the email address of a user defined in the target or a question, the user being used as an actor.

**Comment by titidelaff** [comment]
Will it be added to the next update?

**Comment by titidelaff** [comment]
If not, would someone have a piece of code?
At least give me hints about the files to do it myself

**Comment by btry** [comment]
Hi

the version 2.6.5 is ready for release, without this feature.

If you want to contribute, you can clone the develop branch of this repository (this is the future 2.7.0). 

You should probably start with the file inc/targetticket.class.php. 

I can monitor your progress and give you advices if you create a pull request.

**Comment by titidelaff** [comment]
ty
